core/chains/evm/txmgr/eth_confirmer.go

```diff
@@ -1428,7 +1220,7 @@ func (ec *EthConfirmer) ForceRebroadcast(beginningNonce uint, endingNonce uint,
 	ec.lggr.Infof("ForceRebroadcast: will rebroadcast transactions for all nonces between %v and %v", beginningNonce, endingNonce)
 
 	for n := beginningNonce; n <= endingNonce; n++ {
-		etx, err := findEthTxWithNonce(ec.q, ec.lggr, address, n)
+		etx, err := ec.orm.FindEthTxWithNonce(address, n)
```

Comment:
This (and possible others) no longer have a context passed in (it was implicitly wrapped in the `ec.q`) that will exit early on stop. Can one be added?

Reply:
In the `NewEthConfirmer` the parent context https://github.com/smartcontractkit/chainlink/blob/augustus.BCI-268.part-3-eth-confirmer-sql-refactor/core/chains/evm/txmgr/eth_confirmer.go#L145 isn't actually passed into the `ec.q`. So I think the changes retain the past functionality

Also, note that the purpose of the orm changes is to remove the `ec.q` usage and remove using the Queryer directly.

The orm has it's own internal `q` which will have its own default context so I think we should be good 👍